server/controllers/Messages/inputs.ts

```diff
@@ -0,0 +1,26 @@
+import { MaxLength } from 'class-validator';
+import { IsListEmpty } from './validators/isListEmpty';
+import { ValidateEmailList } from './validators/validateEmailList';
+import { FindDuplicateEmails } from './validators/findDuplicateEmails';
+import { InputType, Field } from 'type-graphql';
+
+// @TODO create custom validator to verify all emails belong to registered users
+// @TODO create custom validator to verify user sending email is authenticated
+
+@InputType()
+export class SendEmailInputs {
+  @Field(() => [String])
+  @IsListEmpty({ message: 'email list cannot be empty' })
+  @ValidateEmailList({ message: 'list contains invalid email' })
+  @FindDuplicateEmails({
+    message: 'list contains one or more duplicate emails',
+  })
+  to: [];
```

Comment:
```suggestion
  to: string[];
```
Shouldn't we specify the type since we know it?  Sorry about the original suggestion, I'm still learning TS and messed up the syntax.